disputer/index.js

```diff
@@ -91,13 +91,17 @@ async function run(address, shouldPoll, pollingDelay, priceFeedConfig, disputerC
 const Poll = async function(callback) {
   try {
     if (!process.env.EMP_ADDRESS) {
-      throw "Bad input arg! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party within your environment variables.";
```

Comment:
By changing this from type `string => object`, the Winston error log will print a stack trace to line 94 of `disputer/index.js` instead of just printing a stack trace to line 121 at the root `catch`